pandas/tools/plotting.py

```diff
@@ -359,7 +359,10 @@ def scatter_matrix(frame, alpha=0.5, figsize=None, ax=None, grid=False,
     density_kwds = density_kwds or {}
 
     # workaround because `c='b'` is hardcoded in matplotlibs scatter method
-    kwds.setdefault('c', plt.rcParams['patch.facecolor'])
+    if 'color' in kwds:
+        pass
+    else:
```

Comment:
nvm, I see that these kwards are *slightly* different. Can you update / add to the comment (and reference the issue).